src/index.js

```diff
@@ -2129,11 +2130,14 @@ function loadLocalFile( obj ) {
 		audioEl.dataset.file = fileBlob.name;
 		audioEl.dataset.title = parsePath( fileBlob.name ).baseName;
 
-		// load and play
-		loadFileBlob( fileBlob, audioEl, true )
-			.then( url => mm.fetchFromUrl( url ) )
-			.then( metadata => addMetadata( metadata, audioEl ) )
-			.catch( e => {} );
+		try {
+			await loadFileBlob( fileBlob, audioEl, true );
+			// Maybe do this parallel?
+			const metadata = await parseBlob( fileBlob );
```

Comment:
I think we could do it parallel, yes.

Reply:
I thought you mean running `parseBlob()` in parallel with `loadFileBlob()` here, since the former don't depend on the latter anymore.